sdk/logs/src/main/java/io/opentelemetry/sdk/logs/LogRecordProcessor.java

```diff
@@ -52,8 +53,10 @@ static LogRecordProcessor composite(Iterable<LogRecordProcessor> processors) {
    * Called when a {@link Logger} {@link LogRecordBuilder#emit()}s a log record.
    *
    * @param logRecord the log record
+   * @param context the context set via {@link LogRecordBuilder#setContext(Context)}, or {@link
+   *     Context#current()} if not explicitly set
    */
-  void onEmit(ReadWriteLogRecord logRecord);
+  void onEmit(ReadWriteLogRecord logRecord, Context context);
```

Comment:
what do you think of switching the arg order to match `SpanProcessor.onStart()`?

we are also passing `Context` first to all of the Instrumenter API methods.

Reply:
Love consistency. Good catch.